.ci/scripts/test.sh

```diff
@@ -37,4 +37,4 @@ fi
 # Run the tests
 export OUT_FILE="build/test-report.out"
 mkdir -p build
-gotestsum --format testname --junitfile "build/junit-${GO_VERSION}.xml" -- ./...
+gotestsum --format testname --junitfile "build/junit-${GO_VERSION}.xml" -- -tags integration -run TestHost_FQDN$ ./... 
```

Comment:
This particular `-tags integration` behaves differently when running on MacOS,

For instance, any builds in the main branch run the tests

<img width="1096" alt="image" src="https://user-images.githubusercontent.com/2871786/215081365-c87a986e-d53e-45d3-a1f0-1ac815274043.png">

but this change does not run anything on MacOS

<img width="1199" alt="image" src="https://user-images.githubusercontent.com/2871786/215081461-203bd604-2b99-459e-92b4-df57c9253ee2.png">


I guess that's only for testing purposes, if so, please forgive me :)

but

Reply:
that is correct, the tests enabled by the `integration` build flag are only for linux

Reply:
If so, then `macos` will need to use a different script, otherwise it will not run any tests